code/src/components/subintro/Subintro.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import Subintro from "@/components/subintro/Subintro";
+import subintroData from "@/components/subintro/Subintro.constants";
+jest.mock("@/assets/images/subintro/moped-icon.jpg", () => {});
+jest.mock("@/assets/images/subintro/box-icon.jpg", () => {});
+jest.mock("@/assets/images/subintro/percent-icon.jpg", () => {});
+jest.mock("@/assets/images/subintro/clock-icon.jpg", () => {});
```

Comment:
You can remove these imports now

Reply:
> You can remove these imports now

I guess he should to try run tests also after rebase with our rule for assets and then remove imports

Reply:
Refactored the test. Now it passes locally, without those mocks.